LauncherInjector/main.cpp

```diff
@@ -224,6 +227,17 @@ bool LoadNorthstar()
 	}
 
 	((bool (*)())Hook_Init)();
+	if (!noLoadPlugins)
+	{
+		Hook_Init = GetProcAddress(hHookModule, "LoadPlugins");
+		if (!hHookModule || Hook_Init == nullptr)
+		{
+			printf("Failed to get function pointer to LoadPlugins of Northstar.dll\n");
+			return false;
+		}
+		((bool (*)())Hook_Init)();
+	}
+
```

Comment:
This doesn't need to be changed now, but for the refactor, we should stop re-using variables like this; it's unnecessarily makes the code less readable and more error-prone.